.travis.yml

```diff
@@ -2,7 +2,6 @@ language: php
 
 php:
   - 5.3
-  - 5.4
```

Comment:
I wanted to leave 5.3 for version `0.4` as it's min. version required by Symfony 2.x